modules/ROOT/pages/apikit-graphql-api-implementation.adoc

```diff
@@ -0,0 +1,525 @@
+
+In the following example, the type *Query* defines the list of queries that the service exposes.
+
+----
+...
+type Query {
+    # Looks for a user by ID
+    getUser(id: ID!): User
+    # Looks for a post by ID
+    getPost(id: ID!): Post
+    # Returns the list of all users
+    allUsers: [User]
+    # Returns the list of all posts
+    allPosts: [Post]
+}
+...
+----
+
+=== Retrieve All the Posts
+
+This use case makes a simple request for the data of each existing post.
+ 
+==== Make a Query
+
+----
+{
+   allPosts {
+       id
+       title
+       body
+   }
+}
+----
+
+==== Trigger a Request
+
+----
+curl --location --request POST 'http://localhost:8081/graphql' \
+--header 'Content-Type: application/json' \
+--data-raw '{"query":"{allPosts{id title body}}","variables":{}}'
+----
+
+==== Get a Response
+
+----
+{
+   "data": {
+       "allPosts": [
+           {
+               "id": "88",
+               "title": "tellus nisi eu orci mauris lacinia sapien quis libero",
+               "body": "Praesent id massa id nisl venenatis lacinia. Aenean sit amet justo. Morbi ut odio.\n\nCras mi pede, malesuada in, imperdiet et, commodo vulputate, justo. In blandit ultrices enim. Lorem ipsum dolor sit amet, consectetuer adipiscing elit.\n\nProin interdum mauris non ligula pellentesque ultrices. Phasellus id sapien in sapien iaculis congue. Vivamus metus arcu, adipiscing molestie, hendrerit at, vulputate vitae, nisl."
+           },
+           {
+               "id": "89",
+               "title": "mauris lacinia sapien quis libero",
+               "body": "Sed ante. Vivamus tortor. Duis mattis egestas metus.\n\nAenean fermentum. Donec ut mauris eget massa tempor convallis. Nulla neque libero, convallis eget, eleifend luctus, ultricies eu, nibh.\n\nQuisque id justo sit amet sapien dignissim vestibulum. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Nulla dapibus dolor vel est. Donec odio justo, sollicitudin ut, suscipit a, feugiat et, eros."
+           },
+           ...
+           {
+            "id": "87",
+               "title": "nam tristique",
+               "body": "Proin eu mi. Nulla ac enim. In tempor, turpis nec euismod scelerisque, quam turpis adipiscing lorem, vitae mattis nibh ligula nec sem.\n\nDuis aliquam convallis nunc. Proin at turpis a pede posuere nonummy. Integer non velit.\n\nDonec diam neque, vestibulum eget, vulputate ut, ultrices vel, augue. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Donec pharetra, magna vestibulum aliquet ultrices, erat tortor sollicitudin mi, sit amet lobortis sapien sapien non mi. Integer ac neque."
+           }
+       ]
+   }
+}
+----
+
+=== Retrieve All Users with Data Loader
+
+This query requests the data of all users, including their address information. In terms of implementation, this example uses a Data Loader to fetch the addresses of each user in only one request.
+
+==== Make a Query
```

Comment:
Make OK? Submit?

Reply:
"Pose a Query" seems to be a better option. 
Should this title be consistent with the title in line 113?
Let's wait for the dev team comments.

Reply:
It has been decided to keep "Make a Query". Thanks!